workflows/rnaseq/main.nf

```diff
@@ -787,6 +787,385 @@ workflow RNASEQ {
+        from QUANTIFY_STAR_SALMON.out.lengths_gene
+        from QUANTIFY_STAR_SALMON.out.counts_gene_length_scaled
+        from QUANTIFY_STAR_SALMON.out.counts_gene_scaled
+        from QUANTIFY_STAR_SALMON.out.tpm_transcript
+        from QUANTIFY_STAR_SALMON.out.counts_transcript
+        from QUANTIFY_STAR_SALMON.out.lengths_transcript
+        from QUANTIFY_STAR_SALMON.out.merged_gene_rds
+        from QUANTIFY_STAR_SALMON.out.merged_gene_rds_length_scaled
+        from QUANTIFY_STAR_SALMON.out.merged_gene_rds_scaled
+        from QUANTIFY_STAR_SALMON.out.merged_transcript_rds
+    }
+
+    "${params.aligner}" {
+        defaults enabled: params.save_align_intermeds || params.save_umi_intermeds
+
+        from SAMTOOLS_SORT.out.bam
+        'umitools/log' {
+            from UMITOOLS_PREPAREFORSALMON.out.log
+        }
+        from UMITOOLS_PREPAREFORSALMON.out.bam
+
+        from BAM_SORT_STATS_SAMTOOLS.bam
+        from BAM_SORT_STATS_SAMTOOLS.bai
+
+        'samtools_stats' {
+            from BAM_SORT_STATS_SAMTOOLS.stats
+            from BAM_SORT_STATS_SAMTOOLS.flagstat
+            from BAM_SORT_STATS_SAMTOOLS.idxstats
+        }
+    }
+
+    "${params.aligner}" {
+        'umitools' {
+            // from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.tsv_edit_distance
+            // from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.tsv_per_umi
+            // from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.tsv_umi_per_position
+        }
+        from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.bam, enabled: params.save_align_intermeds || params.save_umi_intermeds
+        from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.bai, enabled: params.save_align_intermeds || params.save_umi_intermeds
+
+        'samtools_stats' {
+            from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.stats
+            from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.flagstat
+            from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_TRANSCRIPTOME.out.idxstats
+        }
+    }
+
+    //
+    // General alignment options
+    //
+    "${params.aligner}" {
+        'umitools' {
+            // from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.tsv_edit_distance
+            // from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.tsv_per_umi
+            // from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.tsv_umi_per_position
+        }
+        from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.bam, enabled: params.save_align_intermeds || params.with_umi || params.save_umi_intermeds
+        from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.bai, enabled: params.save_align_intermeds || params.with_umi || params.save_umi_intermeds
+
+        'samtools_stats' {
+            from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.stats
+            from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.flagstat
+            from BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.idxstats
+        }
```

Comment:
Why nested?

Reply:
that is the expected directory structure